web/src/components/views/Transform.vue

```diff
@@ -55,17 +60,38 @@ export default {
     txTypeOrNull(tx) {
       if (tx && 'transform_type' in tx) return tx.transform_type;
       return null;
-    }
+    },
+    async loadPCAData (csv) {
+      const pcaData = await CSVService.getPlot(csv, 'pca');
```

Comment:
IMO, this load should go through Vuex.

I consider the PCA of <dataset_id> as part of global state, and vuex should handle what conditions cause PCA to get reloaded.  Putting the watched property on transformed below mixes state logic between the store and the component.  If we reuse PCA (which I'm fairly confident we will) then we have to duplicate the knowledge that "PCA state depends on transformed state".  If the relationship changes, we have to fix it in more than one place. 

In general, using REST services from within a Vue component throws flags for me (as is using watchers on vuex state).  I was sloppy with the image URL because we never intended to keep it and it would be removed as soon as PCA was ready.  I can remove that image in another PR.

After that long-winded digression though, it isn't really a big deal for this PR.  This code is easy to understand, and I think graph state management is a whole other concern that can be addressed in a separate PR because I have an idea how to do it and I'd like feedback from the group because my way may not be the best way.

Reply:
Can you file an issue for this maybe? I am inclined to do it right in this PR, but if you have a more general fix that requires some discussion/education, maybe it's better in a separate PR.

Reply:
Thanks @subdavis, I see #44 now.